test/test.jquery-xblock.js

```diff
@@ -352,4 +352,16 @@ describe('jquery-xblock', function() {
 
     });
 
+    describe('send-runtime-notifications', function() {
+        it("Should send notifications to registered events.", function(done) {
+            $.xblock.dispatcher.bind('test_event', function(event, data) {
+                // Objects are only ever equal to themselves.
+                expect(event.target).to.equal($.xblock.dispatcher[0]);
+                expect(data['test_stuff']).to.equal(5);
+                done();
+            });
+            $.xblock.getRuntime().notify('test_event', {'test_stuff': 5})
```

Comment:
@Kelketek <drag_mode> technically it tests that notify always send `test_event` event with `{test:stuff:5}` content, </drag_mode>. So if it could be extended to at least two events with different data it would be cool. Anyway, one test is infinitely better than no tests (and two tests only two times better than one :)), so not insisting on this.